pkg/api/filter/auth/keystone.go

```diff
@@ -93,7 +93,6 @@ func (k *Keystone) setPolicyContext(r tokens.GetResult) error {
 	}
```

Comment:
Should we change `ExtractProject` method to `ExtractTenant`?

Reply:
No, we cann't. The `ExtrackProject` is one method of third library whose name is gohpercloud. There isn't any method named `ExtractTenant`

Reply:
Got it.